src/StreamWrapper/IncludeInterceptor.php

```diff
@@ -82,33 +87,43 @@ public static function enable(): void
         }
         stream_wrapper_unregister('file');
         stream_wrapper_register('file', __CLASS__);
+        self::$isEnabled = true;
     }
 
     public static function disable(): void
     {
         stream_wrapper_restore('file');
+        self::$isEnabled = false;
+    }
+
+    public static function isEnabled(): bool
```

Comment:
I'm not really a fan of adding a property, and a getter for it, just so we can check the internal workings in a test.

Reply:
i removed the property, i now just try to fetch a file and confirm it still intercept it. cheers